src/server/server_manager.rs

```diff
@@ -171,6 +171,7 @@ impl ServerManager {
                         }
                     };
                 }
+                _ => {}
```

Comment:
The `NetEvent::Accepted` must be used in place of `NetEvent::Connected` for servers. `NetEvent::Connected` is unrachable for servers.